fml/src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java

```diff
@@ -42,16 +44,38 @@ public ItemSmeltedEvent(EntityPlayer player, ItemStack crafting)
     }
 
     public static class PlayerLoggedInEvent extends PlayerEvent {
-        public PlayerLoggedInEvent(EntityPlayer player)
```

Comment:
That would be listening to the event, not firing it. Only Forge should actually post the event to the bus.